### PR TITLE
v0.9.4 release

### PR DESCRIPTION
**Pull request by namhyung**
This is a minor release with following changes:

 * dynamic tracing update (aarch64)
 * python3/luajit scripting support
 * task graph/info support
 * and many bug fixes and improvements

Signed-off-by: Namhyung Kim <namhyung@gmail.com>

### CODE REVIEW

**Comment by honggyukim** [comment]
We can also add some tui changes.
- add `s` key in tui report mode. (Sort by the next column in report)
- reuse `--report` option to start with report mode

**Comment by namhyung** [comment]
Do we have PRs for them?

**Comment by honggyukim** [comment]
I put the PR numbers at the previous comment.

**Comment by namhyung** [comment]
Ah, you just want to add description of the changes.  The PRs are already committed.

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+  experimental support for aarch64
+
+* script update
+  luaJIT (lua 5.1) support by Byeonggon
+  python3 support
+
+* build change
+  update configure script for better compatibility
+  handle common cross compile settings
+
```
Comment (honggyukim) [inline NEWS]: Maybe we can add the following description.
```suggestion

* tui change
  add 's' key to sort by the next column in tui report mode
  use --report option to start tui with report mode

```

Comment (namhyung) [reply NEWS]: Done

**Pull request merged**